src/core/site.jl

```diff
@@ -73,7 +115,17 @@ struct Site
         if length(invalid_args) > 0
             error("Invalid argument values: $(invalid_args)")
         end
+        if !isnothing(renewable_electricity_max_pct) && (renewable_electricity_min_pct > renewable_electricity_max_pct)
+            error("renewable_electricity_min_pct must be less than renewable_electricity_max_pct")
```

Comment:
Should this error be thrown?

Reply:
@NLaws are you suggesting we use `throw(@error " ") `? If so I agree. And @hdunham I think the error should be "less than or equal to..."

Reply:
added "or equal to" and also changed all throw(@error "") to error("") for consistency